Gems/Atom/RPI/Code/atom_rpi_tests_files.cmake

```diff
@@ -18,14 +18,10 @@ set(FILES
     Tests/Common/RPITestFixture.cpp
     Tests/Common/RPITestFixture.h
     Tests/Common/SerializeTester.h
-    Tests/Common/TestUtils.h
-    Tests/Common/TestFeatureProcessors.h
-    Tests/Common/RHI/Factory.cpp
-    Tests/Common/RHI/Factory.h
-    Tests/Common/RHI/Stubs.cpp
-    Tests/Common/RHI/Stubs.h
     Tests/Common/ShaderAssetTestUtils.cpp
     Tests/Common/ShaderAssetTestUtils.h
+    Tests/Common/TestUtils.h
```

Comment:
Seems weird to me that TestUtils.h wasn't moved to the atom_rpi_test_utils_files.cmake file... just wanted to make sure this was intentional.

Reply:
It's intentional. I started by moving *everything* in Tests/Common into atom_rpi_test_utils_files, but I decided to back off from that to just the minimal set of files necessary to use the harness from another gem. More files can always migrate over time if desired, but I didn't want to just arbitrarily move over things that I didn't know for a fact were useful.